osf_tests/test_preprint_summary.py

```diff
@@ -19,11 +19,33 @@ def preprint_provider():
 def preprint(preprint_provider):
     return PreprintFactory._build(PreprintService, provider=preprint_provider)
 
+
+@pytest.fixture()
+def yesterday_at_midnight():
+    return timezone.now().replace(hour=0,
```

Comment:
Let's return a dictionary `preprint_date` and `run_date` for each of these fixtures. If you need, you could also do `expected_result`. Instead of basing on now, just use specific dates for each. like 8/25/2017 at 11:59 UTC for the preprint and 8/26/2017 at 12:00 UTC for the run date or whatever. You can replicate these tests but with absolute dates and we'll be set up for more interesting tests later. Then this won't have a way of being probabilistic any more.